nightwatch/features/event-reconciliation/reconciled.feature

```diff
@@ -0,0 +1,151 @@
+      | Prebookings            | 0    |
+      | Availability           | 1000 |
+      | Scheduled incoming     | 0    |
+      | Scheduled outgoing     | 0    |
+      | Unexpected incoming    | 0    |
+      | Unexpected outgoing    | 0    |
+    When I submit the following movements:
+      | MO In/MO Out | Location | MO Ref. | MO Date | MO Type | CID Person ID |
+      | In           | oneman   | 111     | now     | Removal | 1234          |
+    Then The Centre "one" should show the following under "Male":
+      | Contractual Capacity   | 1000 |
+      | Occupied               | 0    |
+      | Beds out of commission | 0    |
+      | Contingency            | 0    |
+      | Prebookings            | 0    |
+      | Availability           | 999  |
+      | Scheduled incoming     | 1    |
+      | Scheduled outgoing     | 0    |
+      | Unexpected incoming    | 0    |
+      | Unexpected outgoing    | 0    |
+    And I submit the following "check in" event:
+      | centre      | one  |
+      | timestamp   | now  |
+      | cid_id      | 1234 |
+      | person_id   | 12   |
+      | gender      | m    |
+      | nationality | abc  |
+    Then The Centre "one" should show the following under "Male":
+      | Contractual Capacity   | 1000 |
+      | Occupied               | 0    |
+      | Beds out of commission | 0    |
+      | Contingency            | 0    |
+      | Prebookings            | 0    |
+      | Availability           | 1000 |
+      | Scheduled incoming     | 0    |
+      | Scheduled outgoing     | 0    |
+      | Unexpected incoming    | 0    |
+      | Unexpected outgoing    | 0    |
+
+
+  Scenario: Reconciled Check out Event and Movement out does not show as Unexpected out and Scheduled out or affect Availability
+    Given The Centre "one" should show the following under "Male":
+      | Contractual Capacity   | 1000 |
+      | Occupied               | 0    |
+      | Beds out of commission | 0    |
+      | Contingency            | 0    |
+      | Prebookings            | 0    |
+      | Availability           | 1000 |
+      | Scheduled incoming     | 0    |
+      | Scheduled outgoing     | 0    |
+      | Unexpected incoming    | 0    |
+      | Unexpected outgoing    | 0    |
+    When I submit the following movements:
+      | MO In/MO Out | Location | MO Ref. | MO Date | MO Type | CID Person ID |
+      | Out          | oneman   | 111     | now     | Removal | 1234          |
+    Then The Centre "one" should show the following under "Male":
+      | Contractual Capacity   | 1000 |
+      | Occupied               | 0    |
+      | Beds out of commission | 0    |
+      | Contingency            | 0    |
+      | Prebookings            | 0    |
+      | Availability           | 1000 |
+      | Scheduled incoming     | 0    |
+      | Scheduled outgoing     | 1    |
+      | Unexpected incoming    | 0    |
+      | Unexpected outgoing    | 0    |
+    And I submit the following "check in" event:
```

Comment:
It should do. Why don't you submit a test for it?

On Wednesday, 4 May 2016, Chris Nesbitt-Smith <notifications@github.com>
wrote:

> In nightwatch/features/event-reconciliation/reconciled.feature
> <https://github.com/UKHomeOffice/removals_e2etests/pull/6#discussion_r62001194>
> :
>
> > +      | Unexpected outgoing    | 0    |
> > +    When I submit the following movements:
> > +      | MO In/MO Out | Location | MO Ref. | MO Date | MO Type | CID Person ID |
> > +      | Out          | oneman   | 111     | now     | Removal | 1234          |
> > +    Then The Centre "one" should show the following under "Male":
> > +      | Contractual Capacity   | 1000 |
> > +      | Occupied               | 0    |
> > +      | Beds out of commission | 0    |
> > +      | Contingency            | 0    |
> > +      | Prebookings            | 0    |
> > +      | Availability           | 1000 |
> > +      | Scheduled incoming     | 0    |
> > +      | Scheduled outgoing     | 1    |
> > +      | Unexpected incoming    | 0    |
> > +      | Unexpected outgoing    | 0    |
> > +    And I submit the following "check in" event:
>
> would an update individual allow you to add the detainee without having a
> check in?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/UKHomeOffice/removals_e2etests/pull/6/files/8d568934e372574dcdb57ebb6c0def7d6fba6f57#r62001194>
>